core/util/proxy.py

```diff
@@ -0,0 +1,108 @@
+import requests
+from bs4 import BeautifulSoup
+import time
+
+class main:
+
+    def __init__(self, use=False, verbose=False):
```

Comment:
use framework options. Something like this? self.framework._global_options['autoproxy')

Reply:
Done.